schemas/2.0.0.json

```diff
@@ -703,6 +703,14 @@
                     "type": "object",
                     "additionalProperties": false,
                     "properties": {
+                      "name": {
```

Comment:
looking at the current spec description, in my opinion because we have new fields here, we need schema that validates if only `name` and/or `summary` is added, that `headers` or `payload` are required.

What do you think?

Reply:
Good idea. Adding `dependencies` on both `name` and `summary` so that `payload` is then required. I would say that `headers` remain optional in any cases. Pushing an update.

Reply:
but now, doesn't it mean that if users just want to provide an example of `headers` only and `name`, they are forced to provide `payload` example too?

Reply:
Yes that's what it mean, you're right. Is there any cases we'd like to give named example without any payload? This may lead to inconsistent examples imho

Reply:
named example without any example, no, doesn't make sense, but it makes sense that someone just wants to provide an example of the headers only, and this is what is written in the spec in your proposal, so you would need to adjust the `dependency` keyword, that it is either `payload` or `headers` that must be provided when `name` or `summary` are there

Reply:
Seems legit. However I think I stuck to just putting `payload` as required as I don't know if there's a way to say it's either `payload` or `headers`... Whatever the construction used (dependencies, anyOf, if-then-else) I cannot figure out a way to do that in JSON schema ...

Reply:
@lbroudoux I asked on JSON Schema Slack, maybe the will have an idea

Reply:
@magicmatatjahu proposed something like 
```
items: {
  type: object
  properties: {
    ...payload, name, summary, headers
  },
  anyOf: [
    { required: ['payload'] }
    { required: ['headers'] }
  ]
}
```

you would probably have to test it 😄

Reply:
It seems to work well! Expect an update on this PR in few minutes.